### PR TITLE
Dockerfile for dataset generation

### PR DESCRIPTION
**Pull request by nikhil96sher**
Add Dockerfile for generating dataset locally.

### CODE REVIEW

File: experiments.md [hunk 1]
```diff
@@ -2,7 +2,15 @@
 
 ## Data Generation
 
-TODO: Generate TPC-H
+### Generate Locally (Optional)
+TPC-H (Scale 100, Partition 100)
+```bash
+docker build -t deepola-wake:sigmod2023-data -f dockerfiles/data.Dockerfile .
+docker run -i -v ./resources:/dataset/resources:rw -t deepola-wake:sigmod2023-data
+./data-gen.sh 100 100 /dataset
```
Comment (mIXs222) [inline experiments.md:10]: You could do `docker run -rm ... <command>` so lines 9-10 can be combined into one liner.

File: dockerfiles/data.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+FROM alpine
```
Comment (mIXs222) [inline dockerfiles/data.Dockerfile:1]: Pin a version that works?

**Review by mIXs222 — APPROVED** [review]
LGTM with some suggestions

**Comment by pyongjoo** [comment]
@nikhil96sher I think we can merge this? After this, @mIXs222 will update his scripts to optionally use a larger dataset.

**Pull request merged**